cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -69,8 +70,15 @@ fi
 
 # Share munge key
 echo "Sharing munge key"
-mkdir -p /home/${CLUSTER_USER}/.munge
-cp /etc/munge/munge.key /home/${CLUSTER_USER}/.munge/.munge.key
+mkdir -p ${SHARED_DIRECTORY}/.munge
+cp /etc/munge/munge.key ${SHARED_DIRECTORY}/.munge/.munge.key
+chmod 0600 ${SHARED_DIRECTORY}/.munge
+chmod 0600 ${SHARED_DIRECTORY}/.munge/.munge.key
+
+mkdir -p ${SHARED_DIRECTORY_LOGIN}/.munge
+cp /etc/munge/munge.key ${SHARED_DIRECTORY_LOGIN}/.munge/.munge.key
+chmod 0600 ${SHARED_DIRECTORY_LOGIN}/.munge
+chmod 0600 ${SHARED_DIRECTORY_LOGIN}/.munge/.munge.key
```

Comment:
Nitpick (not too high priority): please avoid code deduplication and use a loop over the shared directories here.

Reply:
Done!